api-consumption/src/main/java/org/zalando/nakadi/service/subscription/StreamingContext.java

```diff
@@ -114,6 +117,11 @@ private StreamingContext(final Builder builder) {
         this.kafkaRecordDeserializer = builder.kafkaRecordDeserializer;
         this.eventTypeCache = builder.eventTypeCache;
         this.featureToggleService = builder.featureToggleService;
+
+        this.userFailedCommitLimit = Optional.ofNullable(getSubscription().getAnnotations())
+          .map(ans -> ans.get(DeadLetterAnnotationValidator.AUTO_DLQ_FAILED_COMMIT_LIMIT))
+          .map(Integer::valueOf)
+          .orElse(null);
```

Comment:
Whitespace looks out of surrounding style here (expected 8 spaces on continuation).